.travis.yml

```diff
@@ -1,6 +1,6 @@
 ---
 
-dist: bionic
+dist: focal
```

Comment:
Is this a required change?

Reply:
Yes, when we update the distribution to focal there is no need to install the latest version of docker separately as it is required to use buildx commands.

Reply:
Okay, I got it, if you don't mind, another separated PR can also be sent to upgrade the Ubuntu distro on Travis CI, as it's not a blocking issue to the Docker build, and the distro will be upgraded anyway.